README.md

```diff
@@ -136,6 +137,21 @@ cd tests
 ./run_test.sh
 ~~~~
 
+## Align reads to the transcriptome assembly and visualize the results in IGV
+If you want to align reads to the transcriptome assembly and visualize the results in IGV, you can use the following commands:
+~~~~
+snakemake align_reads --cores 16 
+snakemake IGV --cores 4
+~~~~
+
+Then you can load your transcriptome file to IGV: Genomes -> Load Genome from File -> select the file *transcriptome.fasta*
+
+Your sorted .bam files will be in the bowtie_alignments folder: 
+~~~~
+bowtie_alignments/{sample}.sorted.bam
+bowtie_alignments/{sample}sorted.bam.bai
```

Comment:
`{sample}sorted.bam.bai` or `{sample}.sorted.bam.bai`?

Reply:
Both types of files (.bam and .bam.bai) will be in the folder, in IGV one only uploads the .bam file but .bam.bai (with same file name) needs to be present in the same folder.

Reply:
I was just pointing to the missing period in `{sample}sorted.bam.bai`

Reply:
Ohh I did not read properly... It is fixed now.

Reply:
👍🏻